src/genie/libs/parser/iosxe/tests/test_show_interface.py

```diff
@@ -17398,6 +17398,257 @@ class TestShowIpInterface(unittest.TestCase):
         IPv4 WCCP Redirect exclude is disabled
     '''}
 
+    golden_parsed_output4 = {
```

Comment:
Unittest style has been changed. now we have command output separately. 
Please find the detail from below page.
https://pubhub.devnetcloud.com/media/pyats-development-guide/docs/writeparser/writeparser.html#test-your-new-parser

`ShowIpInterface` parser class's test is now under below location.
https://wwwin-github.cisco.com/pyATS/genieparser/tree/dev/src/genie/libs/parser/iosxe/tests/ShowIpInterface/cli

Could you please adjust to above new style accordingly? Let us know if any questions.

Reply:
confirmed the change